### PR TITLE
Concluding changes to Shop feature

### PR DESCRIPTION
**Pull request by McRight**

### CODE REVIEW

File: src/components/molecules/modals/ShopModal.js [hunk 1]
```diff
@@ -272,12 +263,13 @@ export const ShopModal = ({
...
   const [validEmail, setValidEmail] = useState(false)
   const [emailTouched, setEmailTouched] = useState(false)
+  const [buttonDisabled, setButtonDisabled] = useState(false)
 
   const { name, image, price, description, note, maxQuantity, isPhysical } = shopItem || {
     checkout: {},
   }
 
-  console.log({ MODAL_SHOP_ITEM: shopItem })
+  console.log({ shopItem })
```
Comment (Jonathansoufer) [inline src/components/molecules/modals/ShopModal.js:272]: Left overs?

File: src/components/molecules/modals/ShopModal.js [hunk 2]
```diff
@@ -324,6 +314,9 @@ export const ShopModal = ({
     setValidEmail(true)
   }
 
+  const handleEnableButton = () => setButtonDisabled(false)
```
Comment (Jonathansoufer) [inline src/components/molecules/modals/ShopModal.js:317]: Couldn't use only const handleButtonStatus and change the state accordingly?

**Review by Jonathansoufer — APPROVED** [review]
Looks good to me, but left some small comments.

File: src/components/molecules/modals/ShopModal.js (hunk 2 above)
Comment (McRight) [reply src/components/molecules/modals/ShopModal.js:316]: seemed to me more readable this way

**Pull request merged**